backend/program/symlink.py

```diff
@@ -80,7 +75,8 @@ def validate(self):
     def create_initial_folders(self):
         """Create the initial library folders."""
         try:
-            self.library_path = self.settings.host_path.parent / "library"
+            path = self.settings.symlink_path or self.settings.host_path.parent

```

Comment:
Why 2 path variations?

Reply:
Because I'm an idiot and didn't realize that's what host path is for.  The naming conventions are not very descriptive of how the paths are used.  In general I think it's presumptuous to assume that users are running plex or this application in a container.  I much prefer your `rclone_path` and `symlink_path` descriptions.

Reply:
I'm fine with that, it throws everyone for a loop haha. Go ahead and change the names :D

Reply:
Let me know to not break frontend 😅